qemu/tests/cfg/rng_maxbytes_period.cfg

```diff
@@ -0,0 +1,46 @@
+- rng_maxbytes_period:
+    type = rng_maxbytes_period
+    kill_vm = yes
+    virt_test_type = qemu
+    no no_virtio_rng
+    only Linux
+    read_rng_cmd  = "dd if=/dev/hwrng of=/dev/null bs=1024 count=1024"
+    check_rngd_service = "systemctl status rngd"
+    start_rngd_service = "systemctl start rngd"
+    RHEL.6:
+        driver_name = virtio
+        check_rngd_service = "service rngd status"
+        start_rngd_service = "service rngd start"
+    variants:
+        - positive_test:
+            variants:
+                - scenario1:
```

Comment:
what about max_bytes_1k_period_1k, or something else, which can be more meaningful than scenario1/2/3...

Reply:
yeah, that's acceptable. Updated, thanks!